### PR TITLE
Remove pihole-FTL.conf manpage

### PR DESCRIPTION
**Pull request by yubiuser**
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/contributing/).

---
**What does this PR aim to accomplish?:**
Removes the outdated `pihole-FTL.conf` manpage.
FTL's config options change quite often. We document all changes to FTL's config options in `https://https://docs.pi-hole.net/ftldns/configfile/`. Updating them all also in the man pages has been forgotten for over 1 year now and is likely to be forgotten again.  Having them up-to-date online in one place seems sufficient given that Pi-hole is a DNS forwarder.
I added a link to the online resource in the manpage of `pihole-FTL`.

Bildschirmfoto zu 2021-12-29 07-17-20

**How does this PR accomplish the above?:**
Remove the outdated file, update the installer to remove it from existing installations.

### CODE REVIEW

**Review by DL6ER — APPROVED** [review]

**Comment by DL6ER** [comment]
Connected PR is https://github.com/pi-hole/pi-hole/pull/4081

**Comment by DL6ER** [comment]
Test failed because of https://github.com/pi-hole/pi-hole/blob/1b809e4e8ea442cd2a3cc54206429fabd58cce05/test/test_automated_install.py#L354-L357

not being removed. I'm a bit concerned that only the Fedora 34 failed...

**Comment by PromoFaux** [comment]
> I'm a bit concerned that only the Fedora 34 failed...

It's fail-fast, so because it failed the test were cancelled

**Comment by yubiuser** [comment]
But all the debian based-distros did not fail...

**Comment by PromoFaux** [comment]
As soon as _any_ one test fails, any remaining running tests are cancelled. Those that pass before one fails will remain as passed, innit.

**Comment by DL6ER** [comment]
Yes, but they should not have passed at all.

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -1371,15 +1371,20 @@ install_manpage() {
     # Testing complete, copy the files & update the man db
     install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/manpages/pihole.8 /usr/local/share/man/man8/pihole.8
     install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/manpages/pihole-FTL.8 /usr/local/share/man/man8/pihole-FTL.8
-    install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/manpages/pihole-FTL.conf.5 /usr/local/share/man/man5/pihole-FTL.conf.5
+
+    # remvoe previously installed "pihole-FTL.conf" man page
```
Comment (DL6ER) [inline automated install/basic-install.sh:1375]:
```suggestion
    # remove previously installed "pihole-FTL.conf.5" man page
```

**Comment by yubiuser** [comment]
> Yes, but they should not have passed at all.

I could not see  why they passed on debian. `test_automated_install.py` with `test_installPihole_fresh_install_readableFiles` is called there as well. 
The only reason I can think of is that

https://github.com/pi-hole/pi-hole/blob/1b809e4e8ea442cd2a3cc54206429fabd58cce05/test/test_automated_install.py#L219-L220

fails and ` maninstalled` is set to `False`

**Review by DL6ER — DISMISSED** [review]

**Comment by yubiuser** [comment]
Currently leaving this PR unmerged to remind us to investigate this possible bug with the tox test on debian

**Comment by PromoFaux** [comment]
@yubiuser - what is the status of this please?

**Comment by yubiuser** [comment]
No time for investigation yet.

**Review by PromoFaux — APPROVED** [review]

**Pull request merged**